keywords.robot

```diff
@@ -2978,3 +2978,26 @@ Should Contain All
     FOR    ${substring}    IN    @{substrings}
         Should Contain    ${string}    ${substring}
     END
+
+Flash SD Wire
```

Comment:
@WiktorG351 name is misleading because we are not flashing SD Wire, but "with/via" this device

Reply:
@WiktorG351 another issue is that we are missing `[Arguments]` section

Reply:
@artur-rs adressed [here](https://github.com/Dasharo/open-source-firmware-validation/blob/a5528a512d760cae06aa057a36a7ecb3293e5146/keywords.robot#L2982) and [here](https://github.com/Dasharo/open-source-firmware-validation/blob/a5528a512d760cae06aa057a36a7ecb3293e5146/keywords.robot#L2985)